spec/decanter/parser/boolean_parser_spec.rb

```diff
@@ -61,5 +75,33 @@
           .to raise_error(Decanter::ParseError)
       end
     end
+
+    context 'returns true with options for' do
+      trues_with_options.each do |cond|
+        it "#{cond[0]}: #{cond[1]}, option: {#{cond[2]}: #{cond[3]}}" do
+          expect(parser.parse(name, cond[1], true_value: cond[3])).to match({name => true})
+        end
+      end
+    end
+
+    context 'returns false with options for' do
+      falses_with_options.each do |cond|
```

Comment:
Do you need an array for this or could it be just a single assertion? It looks like you arrived to that conclusion by only having one entry in the array. I think you could make this test more explicit if you kept it just to one entry.